pkgs/development/python-modules/tokenizers/default.nix

```diff
@@ -60,19 +61,22 @@ in rustPlatform.buildRustPackage rec {
     hash = "sha256-rQ2hRV52naEf6PvRsWVCTN7B1oXAQGmnpJw4iIdhamw=";
   };
 
-  cargoSha256 = "sha256-BoHIN/519Top1NUBjpB/oEMqi86Omt3zTQcXFWqrek0=";
+  cargoDeps = fetchCargoTarball {
+    inherit src sourceRoot;
+    sha256 = "sha256-BUiF89lw/QCbifGVD50VpKUl/U+0Kz/508nSaCAdi2k=";
```

Comment:
When updating I need to invalidate both hashes first, right?

Reply:
Indeed, unless we define `name`, because then the version will be part of the vendored name and will invalidate because of that. I'll update the PR to use `name`.

Reply:
Added `name`, so that only the hash in `fetchFromGitHub` needs to be invalidated.